tests/helperServer.js

```diff
@@ -66,6 +67,12 @@ module.exports = {
                     callback(null);
                 });
             },
+            function(callback) {
+                cache.init(function(err) {
+                    if (ERR(err, callback)) return;
+                    callback(null);
+                });
+            },
```

Comment:
Do we need a corresponding `cache.finish()` for tests where we start and stop the server many times? It may not matter if we are leaking.